.circleci/workflows.yml

```diff
@@ -509,18 +493,6 @@ workflows:
             - x86_64-linux-self-test
             - wasm-dist-oxidos
 
-
-  # Record the branch commit for all the jobs bors merges commit into.
-  record-branch-commit:
-    when:
-      or:
-        - equal: [<< pipeline.git.branch >>, "main"]
```

Comment:
We need to move these conditions to the `skip` job, otherwise CircleCI will fail on every `main` commit.